faf-commons-data/src/main/java/com/faforever/commons/replay/ReplayDataParser.java

```diff
@@ -167,22 +168,18 @@ private byte[] decompress(byte[] data, @NotNull ReplayMetadata metadata) throws
   @SuppressWarnings("unchecked")
   private void parseHeader(LittleEndianDataInputStream dataStream) throws IOException {
     replayPatchFieldId = readString(dataStream);
-    dataStream.skipBytes(3);
+    String arg13 = readString(dataStream); // always \r\n
 
     String[] split = readString(dataStream).split("\\r\\n");
     String replayVersionId = split[0];
     map = split[1];
-    dataStream.skipBytes(4);
+    String arg23 = readString((dataStream)); // always \r\n and some unknown character
 
-    int numberOfMods = dataStream.readInt();
+    int sizeModsInBytes = dataStream.readInt();
     mods = (Map<String, Map<String, ?>>) parseLua(dataStream);
 
-    int scenarioSize = dataStream.readInt();
-    this.gameOptions = ((Map<String, Object>) parseLua(dataStream)).entrySet().stream()
-      .filter(entry -> "Options".equals(entry.getKey()))
-      .flatMap(entry -> ((Map<String, Object>) entry.getValue()).entrySet().stream())
-      .map(entry -> new GameOption(entry.getKey(), entry.getValue()))
-      .collect(Collectors.toList());
+    int sizeGameOptionsInBytes = dataStream.readInt();
+    this.gameOptions = ((Map<String, Object>) parseLua(dataStream)).entrySet().stream().filter(entry -> "Options".equals(entry.getKey())).flatMap(entry -> ((Map<String, Object>) entry.getValue()).entrySet().stream()).map(entry -> new GameOption(entry.getKey(), entry.getValue())).collect(Collectors.toList());
```

Comment:
leave this wrapped as it is more readable

Reply:
That's the formatter!! 😛

Reply:
Processed in:

- https://github.com/FAForever/faf-java-commons/pull/143/commits/49ddff984653bd82e11962b070d0ec71953b2bce